components/home/content/LogoContent.js

```diff
@@ -0,0 +1,29 @@
+export const logoItems = [
```

Comment:
Just like with the tools content, I would pull these arrays into the files they belong to and just have them as a constant rather than a prop